docs/components/modeler/web-modeler/process-applications.md

```diff
@@ -106,6 +106,16 @@ Note that when you deploy the process application:
 - Linked external forms will be deployed together with the process application.
 - Linked external BPMN and DMN diagrams are _not_ deployed together. They must be deployed separately.
 
+## Milestones
```

Comment:
In the context of a process application, we should only refer to `Versioning`, and avoid mentioning "Milestones", even if, from the implementation perspective, we are still using milestones behind the scenes.
So, I suggest not mentioning milestones at all on this doc page.
Involving here also @HanselIdes and @RomanKostka.

Reply:
Adjusted in my latest commit, but let me know what you think.